windows/CodePushPackage.cs

```diff
@@ -0,0 +1,364 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+using System.IO;
+using System.IO.Compression;
+using System.Threading;
+using System.Threading.Tasks;
+using Windows.Storage;
+using Windows.Storage.Streams;
+using Windows.Web.Http;
+
+namespace CodePush.ReactNative
+{
+    class CodePushPackage
```

Comment:
What do you think about calling this "UpdateManager"? That way we keep the term "Package" out of the new UWP implementation, and I also don't think we need to "CodePush" prefix thanks to the namespace.